README.md

```diff
@@ -63,15 +83,17 @@ automation:
       data_template:
         title: 'Birthday!'
         message: "{{ trigger.event.data.name }} turns {{ trigger.event.data.age }} today!"
-~~~
+```
 
 If you want to have a notification sent to you at a specific time (instead of midnight), you can use a custom templated sensor and a time trigger.
 Create the sensor:
-~~~
-template:
-  - sensor:
-      - name: "Next birthday"
-        state: >
+```yaml
+sensor:
+  - platform: template
+    sensors:
+      next_birthday:
```

Comment:
This should be left as is in master-branch (so that it supports new template style in HA)

Reply:
Not sure if you meant the yaml formatting here, but I changed it back 😅